pubsub/awssnssqs/awssnssqs_test.go

```diff
@@ -33,33 +33,10 @@ import (
 	"gocloud.dev/pubsub/drivertest"
 )
 
-const region = "us-east-2"
-
-func TestOpenTopic(t *testing.T) {
```

Comment:
So this went into drivertest?

Reply:
Yes, these two tests are already covered by conformance tests.